.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,32 @@
+name: Feature
+description: Suggest a new feature in go-ipfs.
+labels:
+  - kind/feature
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Suggest a new feature in go-ipfs (the program). If you'd like to suggest an improvement to the IPFS protocol, please discuss it on [the forum](https://discuss.ipfs.io).
+
+        Issues in this repo must be specific, actionable, and well motivated. They should be starting points for _building_ new features, not brainstorming ideas.
+
+        If you have an idea you'd like to discuss, please open a new thread on [the forum](https://discuss.ipfs.io).
+
+        **Example:**
+
+        > Reduce memory usage of `ipfs cat` (specific) by buffering less in ... (actionable). This would let me run go-ipfs on my Raspberry Pi (motivated).
```

This is the same as the enhancement example which could confuse people. Maybe something like:

```suggestion
        > Add deduplication-optimized chunking of tar files in `ipfs add` (specific) by examining tar headers ... (actionable). This would let me efficiently store and update many versions of code archives (motivated).
```